debian/rules

```diff
@@ -5,7 +5,7 @@ export GOCACHE=$(shell mktemp -d /tmp/gocache-XXXX)
 
 WIN_ARCHS := amd64 arm64
 COMMON_GOFLAGS := -ldflags=-X=github.com/ubuntu/adsys/internal/consts.Version=$(shell dpkg-parsechangelog -S Version) -mod=vendor -tags=withmo
```

Comment:
Do you mind writing a comment here telling that tags should be last (as we will append it to other tags)?